web/src/app/components/GameWindow.tsx

```diff
@@ -80,10 +79,35 @@ export default function GameWindow({roomID, playerID, players, gameText} : GameW
 	}
 	
 
+	useEffect(() => {
+		let counter = 3;
+
+		const countdownInterval = setInterval(() => {
+
+			counter -= 1;
+			setCountdown(counter);
```

Comment:
nitpick but can we do 


`setCountdown(count => count - 1)`

to avoid another counter variable?